generated/python/grpc-google-cloud-logging-v2/google/cloud/grpc/logging/v2/log_entry_pb2.py

```diff
@@ -1,5 +1,5 @@
 # Generated by the protocol buffer compiler.  DO NOT EDIT!
-# source: google/logging/v2/log_entry.proto
+# source: google/cloud/grpc/logging/v2/log_entry.proto
```

Comment:
Technically incorrect, but I'm not sure we can easily fix this or if it's worthwhile. The source is still `google/logging/v2/log_entry.proto`. Can you create a P3 issue to track this?